packages/wrangler/src/dev/dev.tsx

```diff
@@ -149,35 +148,29 @@ function Dev(props: DevProps): JSX.Element {
   );
 }
 
+export interface DirectorySyncResult {
```

Comment:
The types for `tmp.dirSync` call are wrong, so we redefined it here. I'll send a fix upstream soon.